GNNLux/test/layers/conv_tests.jl

```diff
@@ -77,5 +77,10 @@
 
         #TODO test edge
     end
+
+    @testset "SGConv" begin
+        l = SGConv(in_dims => out_dims, relu)
```

Comment:
the second argument to the constructor is not an activation function. 
I suggest running GNNLux tests locally (they are quite fast) and make sure they pass.

Reply:
will do